frontend/src/constants/strings.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+// reusable messages for error handling, validations, null-screens, etc.
+export const ERROR = "Error.";
+export const ERROR_CANCELED =
+  "Can't complete this request. Please notify mds@gov.bc.ca if this problem persists.";
+export const LOADING = "Loading...";
+export const TRY_AGAIN = "Please try again later";
+export const NO_DATA = "No data available";
+export const MAP_UNAVAILABLE = [
+  "The BC Geographic Warehouse (BCGW) map layers used in this application are unavailable at this time.",
+  "Please notify ",
+  <b>pims@gov.bc.ca</b>,
+  " if this problem persists.",
+];
+
+export const UNAUTHORIZED = "You do not have permission to access this site";
+export const UNAUTHORIZED_PAGE = "You do not have permission to access this page";
+
+export const EMPTY_FIELD = "N/A";
+export const EMPTY = "";
+export const ZERO = "0.00";
+export const UNASSIGNED = "Unassigned";
+
+// default coordinates for center of BC
+export const DEFAULT_LAT = 53.7267;
```

Comment:
Probably should extract these to configurable values that allow us to set the default for each user.

Reply:
I'm fine with handling that on a case by case basis, the majority of these are unused at this moment.